spec/controllers/ems_infra_controller_spec.rb

```diff
@@ -688,6 +688,83 @@
     end
   end
 
+  describe "Kubevirt - update" do
+    before do
+      allow(controller).to receive(:check_privileges).and_return(true)
+      allow(controller).to receive(:assert_privileges).and_return(true)
+      login_as FactoryGirl.create(:user, :features => "ems_infra_new")
+    end
+
+    render_views
+
+    it 'creates on post' do
```

Comment:
@masayag Just wondering if we need the 'creates on post' spec at all, since in the UI we disallow creating an Infra Provider of this type, and only allow the user to update the Provider.

Instead of this -
```rb
kubevirt = ManageIQ::Providers::Kubevirt::InfraManager.where(:name => "openshift_with_kubevirt Virtualization Manager").first
```

is it possible to work with a `FactoryGirl` record ?